### DISCUSSION TITLE
Please add option for vertical center third

### DISCUSSION THREAD

**Discussion by ulisse31** [rxhanson/Rectangle#1489] [Ideas]
Hello!
I work with multiple monitors, and one of the them is a dell wide rotated by 90 degress (so the wide size it oriented from top to bottom rather than left to right). I use this monitor for large text, or multiple app windows, so it's handy to split the monitor in thirds vertically. 
While there is an option to "center third" (horizontally) an app, there is no such counterpart to "center third (vertically)" so that 3 windows can fit into a screen. 
It would be nice to have such support 
See attached image
![Screenshot 2024-10-14 at 12 24 45](https://github.com/user-attachments/assets/86b378de-530a-42c5-8ce7-6cb1a0dc153b)

thanks
Andrea

**Comment by rxhanson** [comment]
Although it's not communicated in the UI, the thirds actions will perform the corresponding portrait aligned third on a portrait display, so you can use "Center Third" here as well.

**Reply by ulisse31** [reply]
Yes, confirmed working thatnks.
I reassigned an hotkey to "center half" option, and the clicking the combo 3 times it does align the window to the center third.
I'll close the discussion as this is resolved, thanks
